include/openthread-types.h

```diff
@@ -175,6 +175,25 @@ typedef struct otMeshLocalPrefix
     uint8_t m8[OT_MESH_LOCAL_PREFIX_SIZE];
 } otMeshLocalPrefix;
 
+#define OT_CHANNEL_MASK_ENTRIES_MAX_NUM    2   ///< Maximum number of Channel Mask Entries
+#define OT_CHANNEL_MASK_MAX_SIZE           4   ///< Maximum size of Channel Mask
+
+/**
+  * This structure represents Channel Mask Entry.
+  *
+  */
+typedef struct otChannelMaskEntry
+{
+    uint8_t mChannelPage;
+    uint8_t mMaskLength;
+    uint8_t m8[OT_CHANNEL_MASK_MAX_SIZE];
+} otChannelMaskEntry;
+
+typedef struct otChannelMask
+{
+    otChannelMaskEntry entries[OT_CHANNEL_MASK_ENTRIES_MAX_NUM];
```

Comment:
My thinking is that while Thread supports multiple channel pages, it only defines a single channel page for now.  The API does not have to follow the internal implementation since the structures are not shared.  For example, you could just remove `otChannelMask` and rename `otChannelMaskEntry` to `otChannelMask`.